profiles/common/modules/custom/ecas/includes/ecas.inc

```diff
@@ -1350,6 +1323,13 @@ EOF;
 EOF;
       break;
 
+    case ECAS_WARNING_MESSAGE_TFA_REQUIRED:
+      $value = <<< EOF
+      You just logged in using a one factor authentication method (I.e. Password) when your account requires a stronger authentication.
+      Please reiterate your authentication by choosing a more secured authentication method (I.e. Mobile Phone + SMS).
```

Comment:
```suggestion
      Please renew your authentication by choosing a more secure authentication method (e.g. Mobile Phone + SMS).
```
I wasn't sure about "reiterate", but "renew" is probably easier to understand regardless.